langs/ptbr.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Português-Brasil",
+  "strings": {
+    "SERVERS": "Servidores",
+    "FAVORITES": "Favoritos",
+    "HISTORY": "Histórico",
+    "DIRECT_CONNECT": "Conexão direta",
+    "DISCONNECT": "Desconectar",
+    "SETTINGS": "Configurações",
+    "EXIT": "Sair",
+    "SERVER_BROWSER": "Lista de servidores",
+    "SEARCH": "Pesquisar",
+    "HIDE_FULL": "Esconder cheios",
+    "HIDE_EMPTY": "Esconder vazios",
+    "HIDE_LOCKED": "Esconder trancados",
+    "UPDATE": "Atualizar",
+    "SERVER_NAME": "Nome do servidor",
+    "PLAYERS": "Jogadores",
+    "GAMEMODE": "Modo de Jogo",
+    "PING": "Latência",
+    "WEBSITE": "Site",
+    "FAVORITE_SERVERS": "Servidores favoritos",
+    "LANGUAGE": "Idioma",
+    "ENTER_PASSWORD": "Digite a senha",
+    "CONNECT": "Conectar",
+    "NICKNAME": "Apelido",
+    "VOICE_CHAT": "Chat de voz",
+    "INPUT_DEVICE": "Dispositivo de entrada",
+    "OUTPUT_DEVICE": "Dispositivo de saída",
+    "DEFAULT": "Padrão",
+    "INPUT_VOLUME": "Volume de entrada",
+    "OUTPUT_VOLUME": "Volume de saída",
+    "CONNECTION_FAILED": "Conexão falhou",
+    "LOST_CONNECTION": "Conexão perdida com o servidor",
+    "UNKNOWN_ERROR": "Erro desconhecido",
+    "WRONG_PASSWORD": "Senha incorreta",
+    "DEBUG_NOT_ALLOWED": "O modo de depuração não é permitido neste servidor",
+    "WRONG_BRANCH": "Servidor usa branch de atualização diferente",
+    "WRONG_STABLE_BUILD": "Servidor está executando uma versão diferente do Alt:V",
```

```suggestion
    "WRONG_STABLE_BUILD": "Servidor está executando uma versão diferente do alt:V",
```
It's written as `alt:V` instead of `Alt:V`.